src/test/scala/examples/RerunWithoutElaboratonAndCompileSpec.scala

```diff
@@ -0,0 +1,51 @@
+// See LICENSE for license details.
+
+package examples
+
+import chisel3._
+import chisel3.iotesters.PeekPokeTester
+import org.scalatest.{FreeSpec, Matchers}
+
+/**
+  * This provides an example of how to re-run a verilator compiled simulation without re-running verilator
+  * Currently this method does re-elaborate the circuit
+  */
+class RerunWithoutElaboratonAndCompileSpec extends FreeSpec with Matchers {
```

Comment:
How useful is this? This looks like a hack that won't necessarily be supported later.